### PR TITLE
Update remaining master specs repo references and clean up command instructions.

### PR DESCRIPTION
**Pull request by dnkoutso**
cc @igor-makarov

### CODE REVIEW

**Comment by CocoaPodsBarista** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please include a CHANGELOG entry to credit yourself! <br />
You can find it at <a href="https://github.com/CocoaPods/CocoaPods/blob/master/CHANGELOG.md">CHANGELOG.md</a>.</td>
    </tr>
  </tbody>
</table>

Here's an example of your CHANGELOG entry:

```markdown
* Update remaining master specs repo references.  
  [dnkoutso](https://github.com/dnkoutso)
  [#issue_number](https://github.com/CocoaPods/CocoaPods/issues/issue_number)
```

*note*: There are two invisible spaces after the entry's text.


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by dnkoutso** [comment]
This might require a `CLAide` change as the output looks weird now.

Screen Shot 2019-07-30 at 3 51 22 PM

File: lib/cocoapods/command/lib/lint.rb [hunk 1]
```diff
@@ -22,18 +22,17 @@ def self.options
...
+              'podspec). Multiple platforms must be comma-delimited'],
             ['--private', 'Lint skips checks that apply only to public specs'],
-            ['--swift-version=VERSION', 'The SWIFT_VERSION that should be used to lint the spec. ' \
-             'This takes precedence over the Swift versions specified by the spec or a `.swift-version` file.'],
-            ['--include-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting via :path.'],
+            ['--swift-version=VERSION', 'The `SWIFT_VERSION` that should be used to lint the spec. ' \
+             'This takes precedence over the Swift versions specified by the spec or a `.swift-version` file'],
+            ['--include-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting via :path'],
             ['--external-podspecs=**/*.podspec', 'Additional ancillary podspecs which are used for linting '\
-              'via :podspec. If there are --include-podspecs, then these are removed from them.'],
```
Comment (dnkoutso) [inline lib/cocoapods/command/lib/lint.rb]: removed all `.` from all commands since some were using them and some were not.

Comment (dnkoutso) [reply lib/cocoapods/command/lib/lint.rb]: actually I am going to opt for adding periods everywhere as it looks very weird without them.

Comment (dnkoutso) [reply lib/cocoapods/command/lib/lint.rb]: bah no, CLAide by default has no periods so we are going to be consistent.

File: spec/functional/command/env_spec.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../../spec_helper', __FILE__)
```
Comment (dnkoutso) [inline spec/functional/command/env_spec.rb]: `env_spec.rb` was actually misplaced and was shipping with our sources. This moves it to its correct location.

**Pull request merged**